### ISSUE TITLE
Create new Espresso Tests for Tables's revised top bar

### ISSUE THREAD

**Issue by wbrunette** [opendatakit/tool-suite-X#484]
NOTE: We will allow 3 people to work on this issue

Remember to use "update-jdk11" branch to access revised top bar.

Develop a new set of tests to check for bugs using Android Espresso tests

https://developer.android.com/training/testing/espresso

**Comment by siva224513** [comment]
Hi @wbrunette  @Redeem-Grimm-Satoshi  can you please assign this issue to me

**Comment by Redeem-Grimm-Satoshi** [comment]
@siva224513 When you send in the first PR for the first test then you can work on this

**Comment by Amiynarh** [comment]
Can I also hop in? cc: @Redeem-Grimm-Satoshi

**Comment by wbrunette** [comment]
@Amiynarh I added you

**Comment by Chinex-Boroja** [comment]
Can I work on this too @wbrunette @Redeem-Grimm-Satoshi

**Comment by Chinex-Boroja** [comment]
For the Table's project structure, the `androidTestUitest` folder does not appear on android studio @wbrunette. I can't seem to find the reason why this is happening

**Comment by wbrunette** [comment]
depends on the branch, we are eliminating the UItest flavor. It's in process.

**Comment by Chinex-Boroja** [comment]
> depends on the branch, we are eliminating the UItest flavor. It's in process.

I am on the upgrade-jdk11 branch @wbrunette

**Comment by wbrunette** [comment]
@Chinex-Boroja yes, it's in process. We pushed the upgrade-jdk11 branch as fast as possible and not all the items got completed.
